src/store/modules/tester/codeToPos.js

```diff
@@ -0,0 +1,142 @@
+const ROW1 = [
+  'Escape',
+  'F1',
+  'F2',
+  'F3',
+  'F4',
+  'F5',
+  'F6',
+  'F7',
+  'F8',
+  'F9',
+  'F10',
+  'F11',
+  'F12',
+  'F13',
```

Comment:
Should it is "PrintScreen" ?
![(C0K5PP3VFWM1`0J1_ZS%LT](https://user-images.githubusercontent.com/720475/135191065-ad1a0ee7-38cc-4f19-8f8f-5d20f6246d2f.png)

Reply:
I'm sorry, but Event key and Code are generated by the Browser, not the web app. Those are the reported values from here https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
That's not generated by QMK at all. If you look at QMK it says KC_PSCR, which is print screen, but print screen is windows specific I believe and is basically F13 which is the original code that operating systems remapped to be print screen. So it is actually F13, not print screen. That's just what your operating system interprets it as. It's in the HID spec.